packages/appstore/src/helpers/total-assets-helper.ts

```diff
@@ -0,0 +1,34 @@
+import { useCFDAccounts, useExchangeRate, usePlatformAccounts } from '@deriv/hooks';
+
+export const isRatesLoaded = (
```

Comment:
```suggestion
export const isRateLoaded = (
```

Reply:
@mahdiyeh-deriv in my opinion, that's correct name for function because we are waiting rates for all needed currencies, not just for one currency. Also, it's the first PR and all changes here I need to update 2nd an 3rd PRs as well :) So, if you think we really need to change it I'll do this, but in this case I ask to do this later )